README.md

```diff
@@ -8,11 +8,11 @@ Benchmarks for Rust crypto libraries
 
 |                                              |       *ring*       |       Octavo       |     rust-crypto    | rust-nettle (Nettle) | rust-openssl (OpenSSL) | sodiumoxide (libsodium) | Windows CNG | Mac/iOS Common Crypto |
 |----------------------------------------------|:------------------:|:------------------:|:------------------:|----------------------|:----------------------:|:-----------------------:|:-----------:|:---------------------:|
-| SHA&#x2011;1 & SHA&#x2011;2                  | :white_check_mark: | :white_check_mark: | :white_check_mark: |                      | :white_check_mark:     |                         |             |                       |
+| SHA&#x2011;1 & SHA&#x2011;2                  | :white_check_mark: | :white_check_mark: | :white_check_mark: |                      | :white_check_mark:     | SHA-256/512 only        |             |                       |
 | HMAC (SHA&#x2011;1 & SHA&#x2011;2)           |                    |                    |                    |                      |                        |                         |             |                       |
 | PBKDF2 (SHA&#x2011;1 & SHA&#x2011;2)         | :white_check_mark: |                    | :white_check_mark: |                      | SHA-1 only             |                         |             |                       |
 | AES&#x2011;128&#x2011;GCM & AES&#x2011;256&#x2011;GCM | :white_check_mark: |           | :white_check_mark: |                      |                        |                         |             |                       |
-| ChaCha20&#x2011;Poly1305                     | :white_check_mark: |                    | :white_check_mark: |                      |                        |                         |             |                       |
+| ChaCha20&#x2011;Poly1305                     | :white_check_mark: |                    | :white_check_mark: |                      |                        | Salsa20&#x2011;Poly1305 |             |                       |
```

Comment:
The goal of these benchmarks is to be apples-to-apples comparisons as much as possible. So, let's make a new row for Salsa20-Poly1305 instead of putting it on the ChaCha20-Poly1305 row.